### DISCUSSION TITLE
Release v0.9.5 / 5.51.5

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1147] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This release fixes an incompatibility with windows server 2022 and presumably Windows 21H2 (once its out), it also adds a hand full of useful improvements allowing automatically starting selected boxes at system boot, as well as auto start the box content.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# Changelog

### added
- added clear commands to log submenus [#391](https://github.com/sandboxie-plus/Sandboxie/issues/391)
- added option to disable process termination prompt [#514](https://github.com/sandboxie-plus/Sandboxie/issues/514)
- added "Options/InstantRecovery" setting to sandboxie-plus.ini to use the recovery window instead of the notification pop-up [#988](https://github.com/sandboxie-plus/Sandboxie/issues/988)
- added ability to rename a non-empty sandbox [#1100](https://github.com/sandboxie-plus/Sandboxie/issues/1100)
- added ability to remove a non-empty sandbox 
- added file browser window to SandMan UI to cover the file-view functionality of SbieCtrl [#578](https://github.com/sandboxie-plus/Sandboxie/issues/578)

### Changed
- generic errors in Sbie UI now show the status code as hex and provide a string description when available

### fixed
- fixed "del" shortcut to terminate a process not always working
- fixed group display issue [#1094](https://github.com/sandboxie-plus/Sandboxie/issues/1094)
- fixed issue when using "run sandboxed" on a file that is already located in a sandbox [#1099](https://github.com/sandboxie-plus/Sandboxie/issues/1099)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/0.9.5'>Release v0.9.5 / 5.51.5</a>.</em>

**Comment by mpheath** [comment]
The Sandboxie-Plus install may need a reboot for this version and previous versions. An install, upgrade and uninstall may display the KmdUtil warning (1061) message. The next version may have the fix that has been commited.

I have already recompiled this version into a new installer to test the fix. Installed, upgrade 4 times times and uninstalled. No warning message from KmdUtil with Win10 VM. I cannot use the unsigned installer on my Win7 so I must wait for the official installer like everyone else.

So please try to be patient as v0.9.6 may have the fixed installer. Letting you know that I know, so we all know.

**Reply by DavidXanatos** [reply]
Today evening or so I'll get new signed installers with the fix and update the release.

**Reply by mpheath** [reply]
It seems SbieDrv is still in the services registry and perhaps still in memory. Without the full uninstall with upgrade, SbieDrv becomes an issue. The command `sc query sbiedrv` lists it as stopped, then seems to clear it's entry in the registry. Using `sc query sbiedrv` again gives an enumeration error. So that is a possible hacky way to resolve.

If you can do soon, would be greatly appreciated.

**Reply by DavidXanatos** [reply]
instalers updated

**Reply by mpheath** [reply]
Tested in VM with install, 4 upgrades and uninstall. No KmdUtil warning (1061) messagebox.

On Win7, uninstall previous 0.9.5. Start install of new 0.9.5, upgrade offered, so I did the command `sc query sbiedrv` to clear the registry. Restarted the install and now could do the clean install. Ran Inno Setup Compiler Editor sandboxed which seems OK.

Considered as fixed. Thankyou David!

**Comment by stormline** [comment]
Hello!
After update from 5.51.0 does not start ICQ ! :(
Please fix if possible.

**Reply by isaak654** [reply]
1) ICQ version? 
2) Download page?
3) The program is installed outside Sandboxie and then run sandboxed?
4) What is your Windows version?

**Reply by stormline** [reply]
1: https://i.gyazo.com/c5710dfec7692c57a7bbe5ef3aa4c605.png
2: https://www.icq.com/download
3: yes
4: 10

on v5.51.0 works ok.

**Reply by isaak654** [reply]
Thank you for the report, it will be fixed on the next build: https://github.com/sandboxie-plus/Sandboxie/issues/1161
